plugins/out_stdout/stdout.c

```diff
@@ -206,19 +206,19 @@ static void cb_stdout_flush(struct flb_event_chunk *event_chunk,
         fflush(stdout);
     }
     else {
-        msgpack_unpacked_init(&result);
-        while (msgpack_unpack_next(&result,
-                                   event_chunk->data,
-                                   event_chunk->size, &off) == MSGPACK_UNPACK_SUCCESS) {
-            if (flb_time_pop_from_msgpack(&tmp, &result, &p) != -1 ) {
-                printf("[%zd] %s: [", cnt++, event_chunk->tag);
-                printf("%"PRIu32".%09lu, ", (uint32_t)tmp.tm.tv_sec, tmp.tm.tv_nsec);
-                msgpack_object_print(stdout, *p);
-                printf("]\n");
-            }
+        dec = flb_event_decoder_create((char*)event_chunk->data, event_chunk->size, 0);
```

Comment:
This is an example that each plugin creates decoder.

However,
Should the decoder be a member of `flb_event_chunk` ?
It may be better that fluent-bit core creates/destroys decoder.